src/routers/tasks.py

```
@@ -464,21 +459,19 @@ async def get_task(
     task_inputs = {
         row.input: int(row.value) if row.value.isdigit() else row.value for row in task_input_rows
     }
     templates = [(tt_io.name, tt_io.io, tt_io.requirement, tt_io.template_api) for tt_io in ttios]
     input_templates = [
         (name, template) for name, io, required, template in templates if io == "input"
     ]
-    filled_templates = await asyncio.gather(
-        *[fill_template(template, task, task_inputs, expdb) for name, template in input_templates],
-    )
+    filled_templates = []
+    for name, template in input_templates:
```

Ruff (0.15.20)

[warning] Loop control variable `name` not used within loop body

(B007)